### PR TITLE
feat: Improve docs around rate limiting, mention higher rate limit for key-value store record endpoints

### PR DESCRIPTION
**Pull request by fnesveda**
The documentation around rate limits was a bit confusing, so I expanded it a bit, to make the distinction between global and per-resource rate limits clearer.

I've also added a mention of the increased rate limits on key-value store records.

### CODE REVIEW

**Review by drobnikj — APPROVED** [review]
👍

File: sources/platform/api_v2/api_v2_reference.apib [hunk 1]
```diff
@@ -324,9 +324,27 @@ Here is the table of the most common errors that can occur for many API endpoint
...
 
 All API endpoints limit the rate of requests in order to prevent overloading of Apify servers by misbehaving clients.
-The default rate limit is 30 requests per second and is applied to every API endpoint except the following list
-of endpoints that are rate limited to 200 requests per second:
 
+There are two kinds of rate limits - a global rate limit and a per-resource rate limit.
+
+### Global rate limit
+
+The global rate limit is set to 250 000 requests per minute.
```
Comment (TC-MO) [inline sources/platform/api_v2/api_v2_reference.apib:332]:
```suggestion
The global rate limit is set to _250 000 requests per minute_.
```

File: sources/platform/api_v2/api_v2_reference.apib [hunk 2]
```diff
@@ -324,9 +324,27 @@ Here is the table of the most common errors that can occur for many API endpoint
...
+For [authenticated]((#authentication) requests, it is counted per user,
+and for unauthenticated requests, it is counted per IP address.
+
+### Per-resource rate limit
+The default per-resource rate limit is 30 requests per second per resource, which in this context means a single Actor, a single Actor run, a single dataset, single key-value store etc.
```
Comment (TC-MO) [inline sources/platform/api_v2/api_v2_reference.apib:337]:
```suggestion
The default per-resource rate limit is _30 requests per second per resource_, which in this context means a single Actor, a single Actor run, a single dataset, single key-value store etc.
```

**Review by TC-MO — APPROVED** [review]
Only suggestion would be to use emphasis on number values like the comments I left, other than that LGTM

**Comment by TC-MO** [comment]
sure thing, apologies, it has been a force of habit for me, will do it this way in the future

**Pull request merged**